### DISCUSSION TITLE
Store Absolute URLs into index.html?

### DISCUSSION THREAD

**Discussion by paullombardcartello** [website-scraper/node-website-scraper#514] [Ideas]
Is there a way to have the URLs of resources that go into the index.html scrape to be absolute instead of relative? My use case is slightly different from a normal scrape situation where the resources need to be found relative to where the installation is locally. I'd like the scraped HTML to have resource all absolute if the original source lists them as absolute. 

![image](https://user-images.githubusercontent.com/55300360/192289607-b55fc0cb-390e-41df-aa81-28ecae8f4d27.png)

The original source:

![image](https://user-images.githubusercontent.com/55300360/192289644-ced5d5b9-666e-4ec8-ac55-0fed9d47f251.png)

**Comment by s0ph1e** [comment]
Hey @paullombardcartello 

It should be possible to achieve this with a custom [plugin](https://github.com/website-scraper/node-website-scraper#plugins) with [getReference action](https://github.com/website-scraper/node-website-scraper#getreference). Hope it helps
